packages/appstore/src/components/wallet-content/wallet-cfds-listing.tsx

```diff
@@ -0,0 +1,228 @@
+            {can_get_more_cfd_mt5_accounts && (
+                <GetMoreAccounts
+                    onClick={toggleAccountTypeModalVisibility}
+                    icon='IcAppstoreGetMoreAccounts'
+                    title={localize('Get more')}
+                    description={localize('Get more Deriv MT5 account with different type and jurisdiction.')}
+                />
+            )}
+            {available_dxtrade_accounts?.length > 0 && (
+                <div className='cfd-full-row'>
+                    <Text line_height='m' weight='bold' color='prominent'>
+                        {localize('Other CFDs')}
+                    </Text>
+                </div>
+            )}
+            {available_dxtrade_accounts?.map((account: AvailableAccount) => {
+                const existing_accounts = getExistingAccounts(account.platform || '', account.market_type || '');
+                const has_existing_accounts = existing_accounts.length > 0;
+                return has_existing_accounts ? (
+                    existing_accounts.map((existing_account: TDetailsOfEachMT5Loginid) => (
+                        <TradingAppCard
+                            action_type='multi-action'
+                            availability={selected_region}
+                            clickable_icon
+                            icon={account.icon}
+                            sub_title={account.name}
+                            name={`${formatMoney(existing_account.currency, existing_account.display_balance, true)} ${
+                                existing_account.currency
+                            }`}
+                            description={existing_account.login}
+                            platform={account.platform}
+                            key={`trading_app_card_${existing_account.login}`}
+                            is_wallet={true}
+                            is_wallet_demo={!!wallet_account.is_virtual}
+                        />
+                    ))
+                ) : (
+                    <TradingAppCard
+                        action_type='get'
+                        availability={selected_region}
+                        clickable_icon
+                        icon={account.icon}
+                        name={account.name}
+                        platform={account.platform}
+                        description={account.description}
+                        key={`trading_app_card_${account.name}`}
+                        is_wallet={true}
+                        is_wallet_demo={!!wallet_account.is_virtual}
+                    />
+                );
+            })}
+        </React.Fragment>
+    );
+
+    return (
+        <ListingContainer
+            className='wallet-content__border-reset wallet-content__cfd'
+            title={
+                !is_mobile && (
+                    <div className='cfd-accounts__title'>
+                        <Text size='sm' line_height='m' weight='bold' color='prominent'>
```

Comment:
line-height is m by default

```suggestion
                        <Text size='sm' weight='bold' color='prominent'>
```